runtime/interpreter/decode.go

```diff
@@ -239,7 +239,8 @@ func (d Decoder) decodeStorable() (atree.Storable, error) {
 }
 
 func (d Decoder) decodeString(v string) *StringValue {
-	return NewStringValue(v)
+	// TODO: meter? but string is already decoded
```

Comment:
similarly to above, we don't want to incentive people to try to use storage to get around metering.

Reply:
Agreed, we should meter decoded values. The problem here is that it by the time this function the value constructor is called here, the allocation already happened, in the CBOR library.

Reply:
Pinged Faye to ask how we can get the size of the string from the CBOR stream decoder before calling the string decoding function (`StreamDecoder.DecodeString`)

Reply:
All strings which are decoded are also metered now (not just string values, but also e.g in locations, `StringAtreeValue`, etc)